provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -9,7 +9,9 @@
 from hare_mp.store import ValueProvider
 from hare_mp.types import (ClusterDesc, DiskRef, DList, Maybe, NodeDesc,
                            PoolDesc, PoolType, ProfileDesc, Protocol, Text,
-                           M0ServerDesc, DisksDesc, AllowedFailures, Layout)
+                           M0ServerDesc, DisksDesc, AllowedFailures,
+                           Layout)
+from hare_mp.utils import Utils
```

Comment:
I can't see this module in the patch. Did you forget to add this file to git?

Reply:
Added.